resources/definitions/fdmprinter.def.json

```diff
@@ -1970,6 +1970,21 @@
                             "limit_to_extruder": "top_bottom_extruder_nr",
                             "settable_per_mesh": true
                         },
+                        "speed_ironing":
+                        {
+                            "label": "Ironing Speed",
+                            "description": "The speed at which to pass over the top surface.",
```

Comment:
', when Ironing is enabled' or ', when performing ironing'

By comparison: you describe the jerk explicitly as the jerk 'with which ironing is performed'